packages/astro/components/Code.astro

```diff
@@ -20,7 +22,7 @@ interface Props {
 	 *
 	 * @default "plaintext"
 	 */
-	lang?: BuiltinLanguage | SpecialLanguage | LanguageRegistration;
+	lang?: BuiltinLanguage | SpecialLanguage // | LanguageRegistration;
```

Comment:
Why is that commented?

Reply:
it was causing problems with the lang property. Specifically LanguageRegistration doesn't seem to bring back a string and HTMLAttributes expects a string for lang. I haven't figured out how to fix it yet so I commented it while I look for a solution.

Reply:
`LanguageRegistration` is also giving me an error on line 78-89:
``` js
const highlighter = await getCachedHighlighter({
	langs: [
		typeof lang === 'string'
			? Object.keys(bundledLanguages).includes(lang)
				? lang
				: 'plaintext'
			: lang,
	],
	theme,
	experimentalThemes,
	wrap,
});
```
`Type 'string' is not assignable to type 'LanguageRegistration'.`
But this was before any of my changes.

Reply:
`LanguageRegistration` is also giving me an error on line 78-89:
``` js
const highlighter = await getCachedHighlighter({
	langs: [
		typeof lang === 'string'
			? Object.keys(bundledLanguages).includes(lang)
				? lang
				: 'plaintext'
			: lang,
	],
	theme,
	experimentalThemes,
	wrap,
});
```
`Type 'string' is not assignable to type 'LanguageRegistration'.`
But this was before any of my changes.

Reply:
not sure what's the best here either, let's keep this discussion unresolved for now and see if anyone has an idea